alf/environments/metadrive/map_perception.py

```diff
@@ -0,0 +1,191 @@
+# Copyright (c) 2022 Horizon Robotics and ALF Contributors. All Rights Reserved.
```

Comment:
This file is from the previous PR, right?

Reply:
Yes. Just did a rebase and fixed this.